ansible/roles/virtualbmc-daemon/tasks/main.yml

```diff
@@ -1,4 +1,11 @@
 ---
+- name: Ensure package dependencies are installed
+  package:
+    name: "{{ item }}"
+    state: installed
+  with_items: "{{ vbmcd_packages }}"
```

Comment:
Nit: can we use `loop` here to be more Ansible 2.6-ish?

Reply:
Just an old man stuck in his ways.

Coming right up